blockmanager.go

```diff
@@ -838,12 +838,18 @@ func (b *blockManager) getCheckpointedCFHeaders(checkpoints []*chainhash.Hash,
 	log.Infof("Starting to query for cfheaders from "+
 		"checkpoint_interval=%v", startingInterval)
 
-	queryMsgs := make([]wire.Message, 0, len(checkpoints))
+	// We'll determine how many queries we'll make based on our starting
+	// interval and our set of checkpoints. Each query will attempt to fetch
+	// two checkpoint intervals worth of filter headers. If we have an odd
+	// number of checkpoint intervals to request, then the last query will
+	// only span one checkpoint interval.
+	numQueries := ((len(checkpoints) + 1) / 2) - int(startingInterval+1)/2
```

Comment:
🧐

Reply:
Lol this can be simplified to `(len(checkpoints)-startingInterval+1)/2`.